src/scoreboard/components/Scoreboard/Scoreboard.component.tsx

```diff
@@ -187,6 +187,8 @@ export const Scoreboard: FC<ScoreboardProps> = () => {
             <input type="button" onClick={toggleEditMode} value={editModeScores ? 'Save changes' : 'Oops'} />
             {editModeScores && <input type="button" onClick={() => setEditModeScores(undefined)} value="Cancel" />}
           </div>
+          <br />
+          <br />
```

Comment:
You can just increase the marginTop from the div below.

Reply:
Nice, I used that and also decided that an `hr` would look nice in there as a logical separator